packages/microsite/package.json

```diff
@@ -15,7 +15,9 @@
   "scripts": {
     "prepare": "npm run build",
     "prebuild": "node ./scripts/clean.js",
-    "build": "tsc -p tsconfig.json --skipLibCheck"
+    "build": "tsc -p tsconfig.json --skipLibCheck",
+    "test": "node ../../scripts/test.js --once",
```

Comment:
Would it make more sense to move these `scripts` to the monorepo root and have the script be responsible for running all the packages tests? That's usually how I see Jest setup in a monorepo.

Reply:
Certainly happy to do that. I was considering doing that, but when I've worked with Jest on past projects it was so unbearably slow and false failure prone (the way it does "isolation" is horrifically stateful with any module that has side-effects on import) that I did everything I could to make it easy to run as few tests as possible during dev. I didn't even consider that case for this initial PR, so I have no problem with the naive solution being "just run everything". All the tools in use here are so fast anyway that it may even be good enough long term.

Reply:
The one concern I have with this as I’m trying to make the change is that it may not play nicely with any config difference between packages, as it’ll resolve to a different `CWD`. But I’ll see if I can make it work, I’m not giving up!

Reply:
Right that makes sense. I don't think I'm extending a base config between packages to keep them consistent. That's probably a good idea.